testdata/service/input/service.yaml

```diff
@@ -1,10 +1,22 @@
 apiVersion: v1
 kind: Service
 metadata:
+  creationTimestamp: 2017-02-20T08:40:02Z
   name: myapp
+  namespace: default
+  resourceVersion: "22654527"
+  selfLink: /api/v1/namespaces/default/services/myapp
+  uid: 2bbe0777-f748-11e6-aee4-124f2ed5f410
 spec:
+  clusterIP: 10.0.82.240
```

Comment:
We have 2 separate cases here:
- clusterIP: None. This says as is in Chart.
- clusterIP: <IP>. This is removed from chart, because this will be automatically assigned by cluster.